pkg/client/envconfig.go

```diff
@@ -19,8 +19,9 @@ type Env struct {
 	LoginClientID      string `env:"TELEPRESENCE_LOGIN_CLIENT_ID,default=telepresence-cli"`
 	UserInfoURL        string `env:"TELEPRESENCE_USER_INFO_URL,default=https://${TELEPRESENCE_LOGIN_DOMAIN}/api/userinfo"`
 
-	Registry   string `env:"TELEPRESENCE_REGISTRY,default=docker.io/datawire"`
-	AgentImage string `env:"TELEPRESENCE_AGENT_IMAGE,default="`
+	ManagerNamespace string `env:"TELEPRESENCE_MANAGER_NAMESPACE,default=ambassador"`
```

Comment:
Supporting this is probably worth mentioning in the changelog.

Reply:
Perhaps, but it isn't a change really. We've always supported that env using an `os.GetEnv()` in the installer.

Reply:
I remove this change as it's not relevant to this PR anyway.